lib/editor/editor.dart

```diff
@@ -141,21 +141,23 @@ class EditorState extends State<Editor> {
     return calculatedFontSize;
   }
 
-  handleFileInit(FileIDE file) async {
+  handleFileInit() async {
     SharedPreferences prefs = await SharedPreferences.getInstance();
-    String fileContent = file.content;
+    String fileContent = widget.defaultValue;
+    EditorRegionOptions? region = widget.options.regionOptions;
 
     WidgetsBinding.instance.addPostFrameCallback((timeStamp) {
-      handleRegionFields(file);
+      handleRegionFields();
 
-      if (file.hasRegion) {
-        int regionStart = file.region.start!;
-        if (prefs.get(file.id) != null) {
-          regionStart =
-              int.parse(prefs.getString(file.id)?.split(':')[0] ?? '');
+      if (widget.options.regionOptions != null) {
```

Comment:
You're storing `regionOptions` in `region` variable. Use that as then you won't have to use the not-null assertion operator in the line below